app/views/pages/login.html.slim

```diff
@@ -0,0 +1,17 @@
+.container
+  .row
+    .col-md-6.offset-md-3
+      br
+      br
+      br
+      .card
+        .card-header.text-center
+          card.text ログイン・新規登録
+        .card-body
+          .container
+            .form-group
+              button.btn.btn-facebook.btn-block facebook
```

Comment:
__[Goodcheck]__

Misspelling?

GitHub is not Github.
JavaScript is not Javascript.
Facebook is not facebook!


<!-- Comment by [SideCI](https://sideci.com) -->